doc/release/upcoming_changes/27288.improvement.rst

```diff
@@ -0,0 +1,2 @@
+Scalar comparisons between different types such as `np.array(1) == np.array('s')`
+now return a Numpy bool instead of a Python bool. 
```

Comment:
```suggestion
* Scalar comparisons between non-comparable dtypes such as
  `np.array(1) == np.array('s')` now return a NumPy bool instead of
  a Python bool. 
```
Just making it a bullet and minor tweak, thanks.